pom.xml

```diff
@@ -107,6 +103,9 @@ THE SOFTWARE.
     <changelog.url>https://jenkins-ci.org/changelog</changelog.url>
   </properties>
 
+  <!-- Note that the 'repositories' and 'pluginRepositories' blocks below are actually copy-pasted
+       from the Jenkins org pom. This is on purpose to keep jenkins core buildable even if one has
+       *not* defined the specific details in the settings.xml file. -->
```

Comment:
BTW almost every plugin POM [does the same](https://github.com/jenkinsci/maven-hpi-plugin/blob/e63607715c2362f447dedab74489722254ce64e3/hpi-archetype/pom.xml#L60-L71).